lib/nunjucks/views/error-handling/server-error.njk

```diff
@@ -1,3 +1,4 @@
+{%- set assetPath = '/manage-prototype/dependencies/govuk-frontend/govuk/assets' -%}
```

Comment:
Just taking another look.  We've got a layout we're using for Manage Prototype section which I think we should be using here, [that layout sets `assetPath` as you are here](https://github.com/alphagov/govuk-prototype-kit/blob/main/lib/nunjucks/views/manage-prototype/layout.njk#L1-L2) so I'd suggest we remove the explicit `assetPath` and extend `views/manage-prototype/layout.njk` instead.

[You can see this on the 404 page](https://github.com/alphagov/govuk-prototype-kit/blob/main/lib/nunjucks/views/error-handling/page-not-found.njk#L1), I must have missed the error page when I was going through the internal pages.

Reply:
Sounds good to me. Pushed it up 👍

Reply:
@nataliecarey Going to push again as the new layout changes `application.css` to `manage-prototype.css`

There was a **knownPaths** entry that expected `application.css`

```mjs
const knownPaths = {
  '/public/stylesheets/application.css': {
    type: 'text/css',
    contents: fs.readFileSync(path.join(process.cwd(), '.tmp', 'public', 'stylesheets', 'manage-prototype.css'))
  }
}
```